Analyses/src/ReadVirtualDetector_module.cc

```diff
@@ -127,6 +127,7 @@ namespace mu2e {
   class ReadVirtualDetector : public art::EDAnalyzer {
 
     typedef vector<int> Vint;
+    typedef vector<string> Sint;
```

Comment:
name Sint is funny.  Presumably a variation of Vint, but that stands for (Vector of integers) I guess.